Add triggers run on info to multiple config files page (#9224)
circleci/circleci-docs: jekyll/_cci2/set-up-multiple-configuration-files-for-a-project.adoc

= Set up multiple configuration files for a project
:page-layout: classic-docs
:page-liquid:

> :page-description: A step by step guide for setting up multiple configuration files as multiple pipelines and triggers for a project in CircleCI.

Reviewer: How about this:
```suggestion
:page-description: A step by step guide for setting up multiple pipelines, each defined in a separate configuration file, in a CircleCI project.
```

:icons: font
:experimental:

> You can use multiple configuration files in a CircleCI project and set up separate, multiple pipelines and triggers for each one. Each trigger can be configures to initiate a separate pipeline on the specified trigger event. If your existing configuration file is getting too large to manage, consider splitting it into multiple files.

Reviewer: ```suggestion
A CircleCI project can have any number of pipelines, each defined in a separate configuration file. The configuration files can all be stored in the same repo, or even in different repos if you wish. 
Each pipeline can be configured to run on one or many specific trigger events.
If your existing configuration file is getting too large to manage, and it includes several unrelated workflows, consider splitting it into multiple files.
```

> As your repository is set up as a project in CircleCI, a `config.yml` file will exist already. If you do not see `.circleci/config.yml`, check which branch you are on. You may have set this up on a non-default branch during the set up process.

Reviewer: In the new world, a `config.yml` is no longer strictly necessary actually, so I think we want to change this

Reviewer: And in the sentence above, on which I can't comment directly, we're forcing users to create the new YML files inside the `.circelci/` directory of the same repo. In reality now you can define pipelines that have config anywhere (e.g. not only in that subdirectory + not necessarily in that same repo). 

I think we might need to flip all this and make it "pipeline centric" rather than "config centric", if you know what I mean?

> Each configuration file can be set up with a pipeline and trigger so it is important that each configuration file is complete with all necessary elements for the pipeline, for example, `version`, `jobs`, `workflows`, `commands`, `executors`, `orbs` etc.

Reviewer: The old version of this is so misleading and poorly worded that I think it's influencing the new version. I might try something different

Reviewer: Each of your configuration files represents the full set of commands necessary for a pipeline to be executed, so it should be complete with all necessary elements - for example, `version`, `jobs`, `workflows`, `commands`, `executors`, `orbs` etc.

. Following from the previous step, select **Triggers** in the sidebar
+
. Select btn:[Add Trigger].
. Select **GitHub App** in the dropdown menu.
. Select btn:[Next].
. Complete the required form fields:
** Give your trigger a descriptive name.
** Select your project repository from the dropdown.
** Choose your new pipeline from the "Choose config to run" menu.
** Choose a trigger event options from the "Run on" menu.
+
[TIP]
====
Using multiple pipelines and triggers, you can build  different pipelines for different events, For example, you could have:

- A `build-test` pipeline that runs on every push. Select "All pushes".
- A `e2e-tests` pipeline, longer and more costly, that runs once when the PR is taken out of draft. Select "PR marked ready for review".
- A `teardown-env` pipeline that runs when a PR is merged. Select "PR merged".

> 

Reviewer: Can add something like
```suggestion
Alternatively, you can use the Web App UI to trigger manually a pipeline of your choosing
```

For a full list, see the xref:github-trigger-event-options#[GitHub trigger event options] page.

> ====

Reviewer: How about moving this outside of a "tip" and above "prerequisites"? I think it describes the value of using multiple pipelines, so it can entice a reader to continue reading

Author: good idea!
